### PR TITLE
Improve tools#mpv layer

### PR DESCRIPTION
**Pull request by wsdjeg**

### CODE REVIEW

File: autoload/SpaceVim/layers/tools/mpv.vim [hunk 1]
```diff
@@ -6,16 +6,53 @@
 " License: GPLv3
 "=============================================================================
 
+if exists('s:musics_directory')
+  finish
+else
+  let s:musics_directory = '~/Musics'
```
Comment (goetzc) [inline autoload/SpaceVim/layers/tools/mpv.vim:12]: In plural? I think `~/Music` is more of a standard name.

**Comment by wsdjeg** [comment]
The position can be changed in your toml file. the default can not be changed as it has been merged into master branch.
--------------
wsdjeg
***@***.*** commented on this pull request.

File: autoload/SpaceVim/layers/tools/mpv.vim (hunk 1 above)
Comment (goetzc) [reply autoload/SpaceVim/layers/tools/mpv.vim:12]: `~/Music` is standard on macOS and also on FreeDesktop's XDG.
https://cgit.freedesktop.org/xdg/xdg-user-dirs/tree/user-dirs.defaults#n10

**Comment by wsdjeg** [comment]
Ok，I will change it. thanks for your tips.

**Pull request merged**